client/src/components/DisplayTable/DisplayTable.tsx

```diff
@@ -59,6 +62,9 @@ interface _DisplayTableState {
   initial_props: _DisplayTableProps;
   visible_col_keys: string[];
   searches: { [keys: string]: string };
+  dropdown_filter: {
+    [col_keys: string]: LegendItemType[];
```

Comment:
These don't wind up passed to a `LegendList`, shouldn't be coupled to their type.